suite-native/accounts/src/components/AccountsList/AccountsListStakingItem.tsx

```diff
@@ -0,0 +1,51 @@
+import { Account } from '@suite-common/wallet-types';
+import { RoundedIcon } from '@suite-native/atoms';
+import { CryptoAmountFormatter, CryptoToFiatAmountFormatter } from '@suite-native/formatters';
+
+import { AccountsListItemBase } from './AccountsListItemBase';
+
+type AccountsListStakingItemProps = {
+    account: Account;
+    stakingCryptoBalance: string;
+    onPress: () => void;
+
+    hasBackground?: boolean;
+    isFirst?: boolean;
+    isLast?: boolean;
+};
+
+export const AccountsListStakingItem = ({
+    account,
+    onPress,
+    stakingCryptoBalance,
+    hasBackground,
+    isFirst,
+    isLast,
```

Comment:
nit: `onPress, hasBackground, isFirst, isLast` does not have to be explicitly named here. Instead spreaded `...otherProps` value could be used and passed in a single line to `AccountsListItemBase`.

Reply:
fixed